acquia/acquia_cms
# ACMS-1107: Update Status report template ACMS version information

**Motivation**
Move hook_preprocess_status_report_general_info() from acquia_claro.theme file to acquia_cms.profile file.

**Proposed changes**
Move hook_preprocess_status_report_general_info() from acquia_claro.theme file to acquia_cms.profile file.

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch
- Head towards /admin/reports/status
- If site installed with ACMS then it should show ACMS version else default

**Merge requirements**
